src/routes/(split)/components/list-item-action/+page.md

```diff
@@ -0,0 +1,117 @@
+</script>
+
+# List Item Action
+
+Used to render multiple items with some information, an icon and actions for each of them.
+
+Mainly used in settings pages to render the properties and the actions alongside.
+
+```html
+<ListItemAction>
+  <img slot="icon" src="{icpLogo}" class="start" />
+  <div class="content">
+    <p>Unlocking</p>
+    <p class="description">No age bonus</p>
+  </div>
+  <button slot="actions" class="secondary">One action</button>
+</ListItemAction>
+```
+
+## Properties
+
+| Property | Description                                                     | Type                    | Default     |
+| -------- | --------------------------------------------------------------- | ----------------------- | ----------- |
+| `testId` | Add a `data-tid` attribute to the DOM, useful for test purpose. | `string` or `undefined` | `undefined` |
+
+## Slots
+
+| Slot name    | Description                                                     |
+| ------------ | --------------------------------------------------------------- |
+| Default slot | The main content of the item.                                   |
+| `logo`       | The logo of the item. Recommended to use 100% width and height. |
+| `actions`    | Slot to add the item actions.                                   |
+
+## Usage Recommendation
+
+### Square Icon
+
+The icon should be square and take all the item height.
+
+The item has a set height of the button height plus some padding. The icon slot is square of the whole item height and the same size for the width.
+
+Therefore, the icon slot should take the whole width and height to be square.
+
+### Full Width Actions Mobile
+
+In mobile, the actions are rendered below the content. It's recommended that the buttons take the full width.
+
+If it's just on button, then stretch it or set the slot to the button. If it's two buttons, make sure they cover the full width together.
+
+## Showcase
+
+The component is within a `div` with background to highlight the component exactly as it is.
+
+**IMPORTANT: The component has no internal padding nor external margin. The spacing with the other components is responsibility of the parent using this component.**
+
+### Showcase Logo
+
+<div class="wrapper">
+  <ListItemAction>
+    <img slot="icon" src={icpLogo} class="start" />
+    <div class="content">
+      <p>Unlocking</p>
+      <p class="description">No age bonus</p>
+    </div>
+    <button slot="actions" class="secondary">One action</button>
+  </ListItemAction>
+</div>
+
+### Showcase Icon
+
+<div class="wrapper">
+  <ListItemAction>
+    <div slot="icon" class="start icon">
+      <IconLockOpen size="24px" />
+    </div>
+    <div class="content">
+      <p>Unlocking</p>
+      <p class="description">No age bonus</p>
+    </div>
+    <button slot="actions" class="secondary">One action</button>
+  </ListItemAction>
+</div>
+
+<style>
+  .wrapper {
+    background-color: var(--card-background);
```

Comment:
in NNS dapp, will it be use with a background too?

Reply:
No. This is just to highlight the element in the page.

Reply:
I see, alright. I don't mind if you remove it because I was a bit unsure about the UI but, just thinking out at loud.